### ISSUE TITLE
train error

### ISSUE THREAD

**Issue by DDxk369** [pp00704831/Stripformer-ECCV-2022-#8]
Hello, when I tried to run this project, I found that although it can be trained, but from the print, it seems that each epoch only uses 3% data training, may I ask why?

log：

Epoch 0, lr 0.0001:   3%|▎         | 60/2103 [00:35<20:07,  1.69it/s, loss=G_loss=0.1060; PSNR=19.7014; SSIM=0.0030]
Validation:   1%|          | 10/1111 [00:02<04:18,  4.26it/s]
Epoch 1, lr 9.99999726118503e-05:   3%|▎         | 60/2103 [00:31<17:45,  1.92it/s, loss=G_loss=0.0749; PSNR=21.0703; SSIM=0.0034]
Epoch 2, lr 9.99998904474312e-05:   3%|▎         | 60/2103 [00:31<17:37,  1.93it/s, loss=G_loss=0.0570; PSNR=23.7320; SSIM=0.0024]
Epoch 3, lr 9.999975350683283e-05:   3%|▎         | 60/2103 [00:31<17:41,  1.93it/s, loss=G_loss=0.0463; PSNR=26.6046; SSIM=0.0010]
Epoch 4, lr 9.999956179020533e-05:   3%|▎         | 60/2103 [00:30<17:32,  1.94it/s, loss=G_loss=0.0446; PSNR=26.1639; SSIM=0.0007]
Epoch 5, lr 9.999931529775897e-05:   3%|▎         | 60/2103 [00:31<17:50,  1.91it/s, loss=G_loss=0.0378; PSNR=26.6630; SSIM=0.0005]
Epoch 6, lr 9.999901402976405e-05:   3%|▎         | 60/2103 [00:30<17:29,  1.95it/s, loss=G_loss=0.0416; PSNR=26.7463; SSIM=0.0002]
Epoch 7, lr 9.999865798655096e-05:   3%|▎         | 60/2103 [00:30<17:33,  1.94it/s, loss=G_loss=0.0361; PSNR=27.4509; SSIM=0.0003]
Epoch 8, lr 9.999824716851013e-05:   3%|▎         | 60/2103 [00:30<17:28,  1.95it/s, loss=G_loss=0.0354; PSNR=27.8255; SSIM=0.0006]
Epoch 9, lr 9.999778157609207e-05:   3%|▎         | 60/2103 [00:31<17:43,  1.92it/s, loss=G_loss=0.0348; PSNR=28.8385; SSIM=0.0003]
Epoch 10, lr 9.999726120980735e-05:   3%|▎         | 60/2103 [00:30<17:34,  1.94it/s, loss=G_loss=0.0337; PSNR=27.6484; SSIM=0.0003]
Epoch 11, lr 9.999668607022663e-05:   3%|▎         | 60/2103 [00:31<17:49,  1.91it/s, loss=G_loss=0.0275; PSNR=29.4621; SSIM=0.0000]
Epoch 12, lr 9.999605615798063e-05:   3%|▎         | 60/2103 [00:31<17:54,  1.90it/s, loss=G_loss=0.0347; PSNR=28.5593; SSIM=0.0007]
Epoch 13, lr 9.99953714737601e-05:   3%|▎         | 60/2103 [00:31<18:01,  1.89it/s, loss=G_loss=0.0283; PSNR=30.1884; SSIM=0.0016]
Epoch 14, lr 9.999463201831592e-05:   3%|▎         | 60/2103 [00:31<18:06,  1.88it/s, loss=G_loss=0.0302; PSNR=29.6451; SSIM=0.0011]
Epoch 15, lr 9.999383779245895e-05:   3%|▎         | 60/2103 [00:31<18:04,  1.88it/s, loss=G_loss=0.0293; PSNR=30.2818; SSIM=0.0027]
Epoch 16, lr 9.999298879706018e-05:   3%|▎         | 60/2103 [00:31<18:06,  1.88it/s, loss=G_loss=0.0259; PSNR=29.3248; SSIM=0.0010]
Epoch 17, lr 9.999208503305064e-05:   3%|▎         | 60/2103 [00:32<18:13,  1.87it/s, loss=G_loss=0.0261; PSNR=31.8037; SSIM=0.0094]
Epoch 18, lr 9.99911265014214e-05:   3%|▎         | 60/2103 [00:31<18:06,  1.88it/s, loss=G_loss=0.0262; PSNR=30.2999; SSIM=0.0063]
Epoch 19, lr 9.999011320322363e-05:   3%|▎         | 60/2103 [00:31<18:02,  1.89it/s, loss=G_loss=0.0241; PSNR=30.9041; SSIM=0.0179]
Epoch 20, lr 9.998904513956852e-05:   3%|▎         | 60/2103 [00:32<18:09,  1.87it/s, loss=G_loss=0.0211; PSNR=32.8062; SSIM=0.0100]
Epoch 21, lr 9.998792231162734e-05:   3%|▎         | 60/2103 [00:31<18:02,  1.89it/s, loss=G_loss=0.0233; PSNR=29.9585; SSIM=0.0228]
Epoch 22, lr 9.998674472063141e-05:   3%|▎         | 60/2103 [00:31<18:02,  1.89it/s, loss=G_loss=0.0230; PSNR=34.3020; SSIM=0.0145]

**Comment by pp00704831** [comment]
Hello,

How many batch sizes do you use?  When you use batch size=8, the number of iterations would be 263: ceil(2103/8) in one epoch, you could check it in the config file.
Otherwise, you should check whether you load the training data correctly.
